recipes/cssmin/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "cssmin" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  sha256: 0649244b431c8d604d443047c66156cf5b0d0677c6beacbdaf40594ee2fd4ea4
+  url: https://github.com/zacharyvoase/cssmin/archive/v{{ version }}.tar.gz
+
+build:
+  noarch: python
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+  number: 0
+  entry_points:
+    - cssmin = cssmin:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  import:
```

Comment:
This should be `imports` not `import`:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-imports

Reply:
Why did the linter not catch that?

Reply:
Interesting... thanks!

Reply:
Good timing with that question. A PR was opened up in the last 24 hours about that very issue: 

https://github.com/conda-forge/conda-smithy/pull/567